happi/cli.py

```diff
@@ -154,11 +155,10 @@ def search_parser(
                 criteria = 'name'
                 value = user_arg
             if criteria in client_args:
-                logger.error(
-                    'Received duplicate search criteria %s=%r (was %r)',
-                    criteria, value, client_args[criteria]
+                raise click.ClickException(
+                    'Received duplicate search criteria %s=%r (was %r)' %
```

Comment:
Time to switch this up to a modern f-string interpolation, e.g.:
`f"Received duplicate search criteria {criteria}={value!r} (was {client_args[criteria]!r})"`